src/common/packager.ts

```diff
@@ -191,28 +195,36 @@ export class Packager {
                     executedStartPackagerCmd = true;
                     return this.monkeyPatchOpnForRNPackager()
                         .then(() => {
-                            let args = ["--port", port.toString()];
+                            let args: any = ["--port", port.toString()];
```

Comment:
But it can contain arrays
[See](https://github.com/Microsoft/vscode-react-native/pull/478/files/4f7b30bcc1df2c472e28b5d22438b013f8edd931#diff-328d76ec2ff0f7b1ba655f2d9de8761dR218)